RDKShell/RDKShell.cpp

```diff
@@ -4546,7 +4553,7 @@ namespace WPEFramework {
                     }
 #ifdef ENABLE_RIALTO_FEATURE
                     rialtoConnector->resumeSession(client);
-                    if(rialtoConnector->waitForStateChange(client,RialtoServerStates::ACTIVE,200))
+                    if(rialtoConnector->waitForStateChange(client,RialtoServerStates::ACTIVE,RIALTO_TIMEOUT_MILLIS))
```

Comment:
The if test is wrong. The correct if test is:

`  if(!rialtoConnector->waitForStateChange(client,RialtoServerStates::ACTIVE, RIALTO_TIMEOUT_MILLIS))`

Reply:
Corrected